library/build.gradle.kts

```diff
@@ -11,6 +10,7 @@ plugins {
 
 dependencies {
   implementation(Libs.kotlin)
+  implementation(Libs.kotlin_reflect)
```

Comment:
why is this necessary? Because of `@UIntXX` annotations?

Reply:
no, not sure, there was a warning for different version
will check dependency tree where this requirement comes from